### ISSUE TITLE
Design the Footer

### ISSUE THREAD

**Issue by aquibbaig** [aquibbaig/club-website#17]

**Comment by rajashree23** [comment]
I want to work on this issue.

**Comment by mishramonalisha76** [comment]
I want to work on this issue.

**Comment by aquibbaig** [comment]
@mishramonalisha76 @rajashree23 Go ahead!

**Comment by alishamohanty** [comment]
@rajashree23 Could you share how you plan to design the footer(If you have mock designs share here)? @aquibbaig How are we planning to maintain the color/theme/pattern of the website? A clear idea of this will help @rajashree23 to design the footer.

**Comment by ankitjena** [comment]
![Screenshot from 2019-04-23 12-17-40](https://user-images.githubusercontent.com/23580602/56560484-304e2800-65c2-11e9-9ac9-67b90bb56f8d.png)

@rajashree23 Something like this.

**Comment by mishramonalisha76** [comment]
/home/monalisha/Downloads/Screenshot from 2019-04-29 20-28-39.png
@ankitjena @aquibbaig @alishamohanty  How is this design?

**Comment by rajashree23** [comment]
![Screenshot from 2019-04-29 20-33-41](https://user-images.githubusercontent.com/33730790/56905873-bab0f300-6abe-11e9-9034-b942e51a8988.png)

**Comment by mishramonalisha76** [comment]
![Screenshot from 2019-04-29 20-28-39](https://user-images.githubusercontent.com/42746736/56905909-d0beb380-6abe-11e9-9f69-511163620bee.png)
@aquibbaig @alishamohanty @ankitjena Is this design okay?

**Comment by rajashree23** [comment]
![Screenshot from 2019-04-29 20-33-41](https://user-images.githubusercontent.com/33730790/56905903-ca303c00-6abe-11e9-9b58-d6143ab630f7.png)

![Screenshot from 2019-04-29 20-34-25](https://user-images.githubusercontent.com/33730790/56905930-dd430c00-6abe-11e9-8246-76edcf2142de.png)

@aquibbaig  @ankitjena @alishamohanty Will these designs do?

**Comment by ganeshpatro321** [comment]
We wouldn't want to have the navbar contents again in the footer.
@rajashree23 The 2nd design looks neat, 
> 
> ![Screenshot from 2019-04-29 20-34-25](https://user-images.githubusercontent.com/33730790/56905930-dd430c00-6abe-11e9-8246-76edcf2142de.png)

We need to match with the color schemes of the home page.(keeping a binary color scheme would be good, discuss this with guy designing home page #45 )

**Comment by ankitjena** [comment]
@mishramonalisha76  @rajashree23 Yeah sure, go with something that looks the best with the rest of the design. Do make a pr. We can check and take a look too. You can always change things later.

**Comment by ankitjena** [comment]
Up for taking!

**Comment by rajashree23** [comment]
Yes, no internet connection issues.
I'm ready to take up

**Comment by rajashree23** [comment]
I'll discuss about the color scheme with the guy designing the homepage.

**Comment by ankitjena** [comment]
> ![Screenshot from 2019-04-23 12-17-40](https://user-images.githubusercontent.com/23580602/56560484-304e2800-65c2-11e9-9ac9-67b90bb56f8d.png)
> 
> @rajashree23 Something like this.
We want something like this, It's simple and has enough detail.

**Comment by mishramonalisha76** [comment]
Okay can we take up from tomorrow I have network issues

**Comment by rajashree23** [comment]
Working on it.

**Comment by aquibbaig** [comment]
@rajashree23 @mishramonalisha76 designs first?

**Comment by aquibbaig** [comment]
Also, the current background image is subject to change

**Comment by rajashree23** [comment]
![Screenshot from 2019-05-11 18-35-10](https://user-images.githubusercontent.com/33730790/57570082-da7bdb80-741b-11e9-8d77-2fc2e5981604.png)
@aquibbaig  this

**Comment by rajashree23** [comment]
This is the color I've included. Icons are not showing working on it. Is the color okay or something else?

![Screenshot from 2019-05-16 00-17-40](https://user-images.githubusercontent.com/33730790/57801009-61a8b680-7770-11e9-9e2c-5f950169a6fc.png)

**Comment by rajashree23** [comment]
![Screenshot from 2019-05-16 00-17-40](https://user-images.githubusercontent.com/33730790/57801413-44281c80-7771-11e9-992b-6ebf73d0f54a.png)



This is the background color. Is this okay  ? Im  working on icons

**Comment by aquibbaig** [comment]
> ![Screenshot from 2019-05-11 18-35-10](https://user-images.githubusercontent.com/33730790/57570082-da7bdb80-741b-11e9-8d77-2fc2e5981604.png)
> @aquibbaig this

@rajashree23 Is something wrong with this shade?

**Comment by rajashree23** [comment]
No the homepage image is light shaded and not so dark bluish..so I kept it like this

**Comment by rajashree23** [comment]
I'll change to this shade only then

**Comment by aquibbaig** [comment]
> Also, the current background image is subject to change

Here!

**Comment by mishramonalisha76** [comment]
Okay we will change the colour according to the background  image . We are now working on icons.

**Comment by aquibbaig** [comment]
@mishramonalisha76 Make a footer as the one mentioned. You can change color and all later

**Comment by mishramonalisha76** [comment]
Okay we will . Icons problem is sorted.

**Comment by rajashree23** [comment]
![Screenshot from 2019-05-16 17-12-10](https://user-images.githubusercontent.com/33730790/57851361-fa394800-77fd-11e9-82e8-38e609538f4e.png)

**Comment by rajashree23** [comment]
this

**Comment by rajashree23** [comment]
![Screenshot from 2019-05-16 17-12-10](https://user-images.githubusercontent.com/33730790/57852401-b72ca400-7800-11e9-8599-214948db8a67.png)

**Comment by aquibbaig** [comment]
Looks good

**Comment by ankitjena** [comment]
@rajashree23 Looks good. Make pr :)))
Then we can suggest changes in code.

**Comment by mishramonalisha76** [comment]
![Screenshot from 2019-05-17 08-25-22(1)](https://user-images.githubusercontent.com/42746736/57900671-6f009680-787f-11e9-991d-a6184d452fa8.png)

**Comment by mishramonalisha76** [comment]
![Screenshot from 2019-05-17 08-25-22(2)](https://user-images.githubusercontent.com/42746736/57900744-bc7d0380-787f-11e9-808a-48c8ca7ee7e7.png)

**Comment by aquibbaig** [comment]
Fine, make a PR!
